src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -2048,6 +2048,90 @@ def reorder_levels():
+        The object must have a datetime-like index (DatetimeIndex, PeriodIndex, or TimedeltaIndex),
+        or the caller must pass the label of a datetime-like series/index to the ``on``/``level`` keyword parameter.
+
+        Parameters
+        ----------
+        rule : DateOffset, Timedelta or str
+            The offset string or object representing target conversion.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            Which axis to use for up- or down-sampling. For Series this parameter is unused and defaults to 0.
+            Must be DatetimeIndex, TimedeltaIndex or PeriodIndex.
+
+            Deprecated since version 2.0.0: Use frame.T.resample(…) instead.
+        closed : {'right', 'left'}, default None
+            Which side of bin interval is closed. The default is 'left' for all frequency offsets except for
+            'ME', 'YE', 'QE', 'BME', 'BA', 'BQE', and 'W' which all have a default of 'right'.
+        label : {'right', 'left'}, default None
+            Which bin edge label to label bucket with. The default is 'left' for all frequency offsets except for
+            'ME', 'YE', 'QE', 'BME', 'BA', 'BQE', and 'W' which all have a default of 'right'.
+        convention : {'start', 'end', 's', 'e'}, default 'start'
+            For PeriodIndex only, controls whether to use the start or end of rule.
+
+            Deprecated since version 2.2.0: Convert PeriodIndex to DatetimeIndex before resampling instead.
+        kind : {'timestamp', 'period'}, optional, default None
+            Pass 'timestamp' to convert the resulting index to a DateTimeIndex
+            or 'period' to convert it to a PeriodIndex. By default, the input representation is retained.
+        on : str, optional
+            For a DataFrame, column to use instead of index for resampling. Column must be datetime-like.
+        level : str or int, optional
+            For a MultiIndex, level (name or number) to use for resampling. level must be datetime-like.
+        origin : Timestamp or str, default 'start_day'
+            The timestamp on which to adjust the grouping. The timezone of origin must match the timezone of the index.
+            If a string, must be one of the following:
+
+            'epoch': origin is 1970-01-01
+            'start': origin is the first value of the timeseries
+            'start_day': origin is the first day at midnight of the timeseries
+            'end': origin is the last value of the timeseries
+            'end_day': origin is the ceiling midnight of the last day
+
+            .. versionadded:: 1.3.0
+        offset : Timedelta or str, default is None
+            An offset timedelta added to the origin.
+        group_keys : bool, default False
+            Whether to include the group keys in the result index when using ``.apply()`` on the resampled object.
+
+        Returns
+        -------
+        pandas.api.typing.Resampler
```

Comment:
```suggestion
        Resampler
```
This should let us link to our own Resampler class.